### PR TITLE
Reset port guids in cleaning

### PR DESCRIPTION
**Pull request by abdallahyas**
This patch adds reseting the port_guid and node_guid parameters
of the ci mahcines vfs. It unloads the mlx5_core and mlx5_ib modules
, This is needed since it seams the ib_kubernetes sometimes fails to do
so and this causes a duplication in vfs guids causing problems in
connectivity. This patch makes sure that the vf's guids are reset.

### CODE REVIEW

**Comment by abdallahyas** [comment]
@moshe010 @adrianchiris

**Comment by adrianchiris** [comment]
i think this should be a post check instead of a fix, as this most likely point to an issue with ib-sriov cni as we expect things to be cleaned properly the pod is teared town.

File: cni_stop.sh [hunk 1]
```diff
@@ -52,6 +52,51 @@ function delete_chache_files {
     rm -rf /var/lib/cni/networks
 }
 
+
+function reset_vfs_guids {
```
Comment (moshe010) [inline cni_stop.sh:75]: is reload the driver will have the same effect?

Comment (abdallahyas) [reply cni_stop.sh:75]: @moshe010 , reloading the driver will set them all to zeros.

Comment (moshe010) [reply cni_stop.sh:75]: so you want to clean the guids this is the best way. Or I don't understand you change...

Comment (abdallahyas) [reply cni_stop.sh:75]: ok, it did not occur to me to try reloading the drivers.

**Comment by abdallahyas** [comment]
> i think this should be a post check instead of a fix, as this most likely point to an issue with ib-sriov cni as we expect things to be cleaned properly the pod is teared town.

@adrianchiris. ok, will add a test case for ib_sriov ci that tests this. but i think this should be added as a clean up step. basically the workflow would be to test if it was probably handled, if not then fail the ci, and then the clean script would reset the drivers. what do you think?

**Comment by abdallahyas** [comment]
> i think this should be a post check instead of a fix, as this most likely point to an issue with ib-sriov cni as we expect things to be cleaned properly the pod is teared town.

Done

**Review by moshe010 — APPROVED** [review]

**Pull request merged**